apache-benchmark/README.md

```diff
@@ -0,0 +1,303 @@
+# Apache Benchmark (ab)
+
+Apache Benchmark is a command-line tool for benchmarking HTTP web servers. It is designed to give you an impression of how your current Apache installation performs.
+
+## Installation
+
+### macOS
+
+```bash
+# Apache Benchmark comes with Apache HTTP Server
+brew install httpd
+
+# Or install just the tools
+brew install apache-ab
+```
+
+### Ubuntu/Debian
+
+```bash
+sudo apt-get update
+sudo apt-get install apache2-utils
+```
+
+### CentOS/RHEL/Fedora
+
+```bash
+# CentOS/RHEL
+sudo yum install httpd-tools
+
+# Fedora
+sudo dnf install httpd-tools
+```
+
```

Comment:
You can assume I will run apache benchmark on MacOS only, then revise installation for MacOS users